src/pg_diffix.c

```diff
@@ -11,14 +11,19 @@
 #include "pg_diffix/utils.h"
 
 #include <limits.h>
+
+#ifdef _WIN64
+#define __WORDSIZE 64
+#endif
+
 #if __WORDSIZE != 64
 #error "This module requires a 64-bit target architecture!"
 #endif
```

Comment:
Looks like we can't catch a 32 bit build on Windows. But it's pretty much extinct anyway...

Reply:
We do catch it, on a 32-bit build `__WORDSIZE` would remain undefined, so the error will be raised and the build will be stopped.

Reply:
Ah, now I see that it's `#ifdef _WIN64`.